### DISCUSSION TITLE
Win10 curl full_node 500 Internal Server Error

### DISCUSSION THREAD

**Discussion by Rimwu** [Chia-Network/chia-blockchain#6555] [Support]
i push below code in cmd
`curl --insecure --cert C:/Users/wu/.chia/mainnet/config/ssl/full_node/private_full_node.crt --key C:/Users/wu/.chia/mainnet/config/ssl/full_node/private_full_node.key -d '{"height":327198}' -H "Content-Type: application/json" -X POST https://127.0.0.1:8555/get_block_record_by_height`
but it doesn't work and response
`500 Internal Server Error

Server got itself in trouble`
Can someone know this situation and how can i fix this bug

**Comment by Rimwu** [comment]
i solved this question,in windows this code should use -d "{}",Double quotes,not apostrophe
so the right code is
`curl --insecure --cert .chia/mainnet/config/ssl/full_node/private_full_node.crt --key .chia/mainnet/config/ssl/full_node/private_full_node.key -d "{\"height\":327198}" -H "Content-Type: application/json" -X POST https://127.0.0.1:8555/get_block_record_by_height`

i hope this can help someone!

**Comment by BrandtH22** [comment]
@Rimwu , based on your previous message it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
